### PR TITLE
CHANGELOG 5.10.1

### PR DESCRIPTION
**Pull request by directionless**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+
+### Table Changes
+
+- Add new macOS SIP config flags ([#8101](https://github.com/osquery/osquery/pull/8101))
+- Added `cloud`_id to `ycloud_instance_metadata` - the vm metadata table for Yandex Cloud ([#8086](https://github.com/osquery/osquery/pull/8086))
+- Allow querying of kernel and filesystem drivers ([#8119](https://github.com/osquery/osquery/pull/8119))
+- Update `es_process_file_events` adding support for open events, and for only triggering on `file_paths` ([#8114](https://github.com/osquery/osquery/pull/8114))
+- Update `firefox_addons` to use rapidjson to parse and don't block on read ([#8089](https://github.com/osquery/osquery/pull/8089))
+- Add some indexing to `block_devices` ([#8037](https://github.com/osquery/osquery/pull/8037))
+- Revert "Add some indexing to `block_devices`" ([#8151](https://github.com/osquery/osquery/pull/8151))
```
[the remark this answers is not in the document]
Comment (directionless) [reply CHANGELOG.md:24]: I went back and forth. I've been asked when I didn't have a PR listed in the changelog. I could combine them into `Add and Revert ...`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,81 @@
...
+- Remove forensicReadFile ([#8085](https://github.com/osquery/osquery/pull/8085))
+- Substitute the TEXT macro with SQL_TEXT in table code ([#8091](https://github.com/osquery/osquery/pull/8091))
+- Use JSON member iterator instead of rescanning ([#8122](https://github.com/osquery/osquery/pull/8122))
+- core: Avoid checking if a file exists before opening ([#8087](https://github.com/osquery/osquery/pull/8087))
+- improvement: Avoid unnecessary string conversions ([#8093](https://github.com/osquery/osquery/pull/8093))
+- watchdog: Use virtual cores to calculate CPU utilization limit ([#8104](https://github.com/osquery/osquery/pull/8104))
+
+### Bug Fixes
+
+- Add new AWS valid regions ([#8110](https://github.com/osquery/osquery/pull/8110))
```
[the remark this answers is not in the document]
Comment (directionless) [reply CHANGELOG.md:40]: Sure. Moved to new.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,81 @@
 # osquery Changelog
 
+<a name="5.10.1"></a>
+## [5.10.1](https://github.com/osquery/osquery/releases/tag/5.10.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.9.1...5.10.1)
+
+Representing commits from 18 contributors! Thank you all.
+
```
[the remark this answers is not in the document]
Comment (directionless) [reply CHANGELOG.md:14]: maybe like that?

[the remark this answers is not in the document]
Comment (Smjert) [reply CHANGELOG.md:14]: Yes, perfect!

**Review by Smjert — APPROVED** [review]

**Pull request merged**